Contents/Code/__init__.py

```diff
@@ -129,6 +129,7 @@ def doCreateXMLFile(menuCall = False):
 	for Channel in Channels:
 		channel = ET.SubElement(root, 'channel', id=str(Channel['id']))
 		ET.SubElement(channel, 'display-name').text = ValidateXMLStr(Channel['name'])
+		ET.SubElement(channel, 'display-name').text = str(Channel['id']))
```

Comment:
This won't work ... "display-name" attribute (line 131) is overwritten with the channel Id (line 132). Might wanna change the attribute name from "display-name" to ex. "channel-id".